src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -16,14 +16,14 @@ You can also reach the moderators of this BEP through our [main discussion forum
 Most core principles of the original BIDS and particulars of BIDS-iEEG specification are adopted
 for this modality as well, though some special considerations and additional fields were added.
 
-Several [example x datasets](https://bids-standard.github.io/bids-examples/#ephys)
+Several [example x datasets](https://bids-standard.github.io/bids-examples/#microephys)
```

Comment:
what does the x in "example x datasets" mean?

Reply:
good catch, I think it was intended to be a placeholder, hence now we know that it should be 

```suggestion
Several [example Microelectrode Electrophysiology datasets](https://bids-standard.github.io/bids-examples/#microephys)
```